config_defaults/subtests/docker_cli/run_volumes.ini

```diff
@@ -2,6 +2,9 @@
 subsubtests = volumes_rw, volumes_one_source
 #: These must be setup prior to testing
 __example__ = host_paths, cntr_paths
+#: Command(s) required to succeede prior to testing (if any)
+#: e.g. setsebool virt_sandbox_use_nfs
```

Comment:
Sorry for not being clear. What I meant was, in your comment you specify "command(s)" which implies that one can specify multiple commands. But it isn't clear to the config-file editor (who might not be familiar with utils.run, and might not want to be) _how_ to specify multiple commands. Comma? ["command1", "command2"]? A hint ("use semicolon to separate multiple shell commands") could steer someone along the right path.